website/src/blogs/2017-06-28-storage-is-dead-long-live-openebs.md

```diff
@@ -0,0 +1,74 @@
+
+Last but not least, DevOps is a culture and a set of practices that, as demonstrated by the story of the crucial book the Phoenix Project, is motivated in part by a realization of how broken, slow, dysfunctional, unfair, and brittle, proprietary IT systems and IT operations traditionally have been. From this perspective, applying infrastructure as code in part to ensure cross team transparency and blameless postmortems and so forth are not just examples of useful techniques — they are manifestations of the values of a better culture.
+
+One strength of clouds like AWS is that they enable you to consume storage in a DevOps friendly way — your orchestration or other automation can deal with the provisioning and management (or can rely on the solutions provided by the cloud provider) so that intent can flow from the application through the infrastructure; in addition, the infrastructure is there to be consumed dynamically, via APIs. Unfortunately, the cloud providers themselves have become the new proprietary, which makes some in the DevOps world a bit uncomfortable as in the past lock-in was used by proprietary vendors to maximize their own prices at the expense of their customers. Kubernetes especially offers a chance to keep the wiring free from the cloud providers, hence providing some hope of avoiding lock-in.
+
+****Uhh, huh — and OpenEBS?****
+
+![Uhh, huh — and OpenEBS?](/images/blog/uhh-huh-openebs.png)
+
+**Uhh huh — data, cloud, DevOps, lock-in and, err, OpenEBS?**
+
+In short, the highest level vision for OpenEBS is one in which the DevOps friendly aspects of running storage in the cloud is now available on-premise and across your clouds, thus freeing you from vendor lock-in much in the way that Kubernetes helps you avoid the risk of locking yourself in through the use of cloud service specific wiring and workflow.
+
+What if your developers and DevOps teams could orchestrate your storage controllers just like other containers? How much more productive could they be?
+
+And what if that set of storage services allowed you to treat your stateful workloads on containers much like the ephemeral containers that have proliferated?
+
+And what if your storage and CIO teams were able to establish policies that governed your data without impeding the agility that was your purpose in moving towards containers and DevOps in the first place?
+
+Last but not least, what if, having tooled your environment to work with Amazon it just worked when you added the on-premise and cross-cloud OpenEBS? Wouldn’t this limit your lock-in with any one cloud vendor?
+
+OpenEBS remains early. Today we are at release 0.3 and are just starting to be used by enterprises like Cap Gemini and others to deliver storage for stateful workloads on containers.
+
+I’m back into the storage industry because I’m sure that OpenEBS has a good chance to allow storage to slip into the background and “just work” to support the incredible boosts in agility delivered by DevOps.
```

Comment:
```suggestion
I’m back into the storage industry because I’m sure that OpenEBS has a good chance to allow storage to slip into the background and “`just work`” to support the incredible boosts in agility delivered by DevOps.
```

Reply:
the quotation is intentional. Backtick is added for codes only, usually